Dockerfiles/Dockerfile-CentOS7

```diff
@@ -10,63 +10,22 @@ RUN \
 RUN \
   yum -y install centos-release-openstack-train && \
   yum -y install epel-release && \
-  yum install -y autoconf automake bison blas-devel bzip2 cmake3 curl-devel flex \
-    git git-lfs gsl-devel grpc-devel grpc-plugins gtest-devel hdf5-devel lapack-devel libtool libxml2-devel libzstd-devel \
+  yum install -y autoconf automake bison blas-devel bzip2 cfitsio-devel cmake3 curl-devel flex \
+    git git-lfs gsl-devel grpc-devel grpc-plugins hdf5-devel lapack-devel libtool libxml2-devel libzstd-devel \
     make openssl-devel protobuf-devel pugixml-devel python3 readline-devel subversion systemd-devel wcslib-devel wget \
     zlib-devel libuuid-devel zfp-devel && \
-  ln -s /usr/bin/cmake3 /usr/bin/cmake && \
-  pip3 install numpy astropy
+  pip3 install numpy astropy && \
+  ln -s /usr/bin/cmake3 /usr/bin/cmake
 
-# Install newer cfitsio as default cfitsio 3.37 does not have a function used by fits2idia 
-RUN \ 
-  source /etc/bashrc && \
-  cd /root && \ 
-  wget http://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-3.47.tar.gz && \
-  tar -xvf cfitsio-3.47.tar.gz && cd cfitsio-3.47 && \
-  ./configure --prefix=/usr/local && make && make install && \
-  cd /root && rm -rf cfitsio-3.47 cfitsio-3.47.tar.gz
-
-# Get geodetic and ephemerides casa-data
-RUN \
-  source /etc/bashrc && \
-  mkdir -p /usr/local/share/casacore/data/ephemerides && \
-  mkdir -p /usr/local/share/casacore/data/geodetic && \
-  svn co https://svn.cv.nrao.edu/svn/casa-data/distro/ephemerides/ /usr/local/share/casacore/data/ephemerides && \
-  svn co https://svn.cv.nrao.edu/svn/casa-data/distro/geodetic/ /usr/local/share/casacore/data/geodetic && \
-  rm -r /usr/local/share/casacore/data/ephemerides/splatalogue.db && \
-  rm -r /usr/local/share/casacore/data/ephemerides/SplatDefault.tbl
-
-# Get libsofa.a for casacore
+# CARTA will have a problem finding the .h files in /usr/include/cfitsio/
 RUN \
   source /etc/bashrc && \
-  wget http://www.iausofa.org/2018_0130_F/sofa_f-20180130.tar.gz && \
-  tar xzf sofa_f-20180130.tar.gz && rm sofa_f-20180130.tar.gz && \
-  cd sofa/20180130/f77/src && make && cp libsofa.a /usr/lib/libsofa.a && \
-  cd /root && rm -rf sofa
+  cp /usr/include/cfitsio/* /usr/include/
```

Comment:
Shouldn't this be solved by our use of find_package in the cmake file?

Reply:
That doesn't work for some reason.
The `find_package` seems to only find the .so files. 
The .so file is found properly at `/usr/lib64/libcfitsio.so`, but the include files are not found. 
For some reason on RHEL, the fitsio include files are located in a seemingly non-standard location of `/usr/include/cfitsio/` whereas on Ubuntu they are found in `/usr/include`.

I guess another solution would be to modify all relevant the carta-backend includes to also search for the standard RHEL /usr/include/cfitsio/ location?

Or alternatively we can just do `cmake .. -DCMAKE_CXX_FLAGS="-I/usr/include/cfitsio"`

Reply:
Or maybe add it into an INCLUDE_DIRECTORIES in CMakeLists.txt?

Reply:
@veggiesaurus Should we add `/usr/include/cfitsio` to the CMakeLists.txt. Maybe on this line?

> INCLUDE_DIRECTORIES(/opt/carta-casacore/include /opt/carta-casacore/include/casacore /opt/carta-casacore/include/casacode /usr/include/cfitsio)